### PR TITLE
[agent][windows] adding note for install command-line options

### PR DESCRIPTION
**Pull request by truthbk**
### What does this PR do?
Adds a note regarding command-line options for the windows install.

### Motivation
Some misunderstanding during testing of new proxy settings.

### CODE REVIEW

File: content/agent/basic_agent_usage/windows.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ The following configuration command line options are available when installing t
 | `PROXY_USER`      | String | If using a proxy, sets your proxy user. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
 | `PROXY_PASSWORD`  | String | If using a proxy, sets your proxy password. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                          |
 
+Note: if a valid `datadog.yaml` is found, that file takes precedence over specified command-line options. In practice this means, most of these options only typically applies to fresh installs. Of course you can always modify the relevant options in the `datadog.yaml`.
```
Comment (cswatt) [inline content/agent/basic_agent_usage/windows.md]: Note: if a valid `datadog.yaml` is found, that file takes precedence over specified command-line options. In practice, this means that most of these options only typically apply to fresh installs. Of course, you can always modify the relevant options in `datadog.yaml`.

**Comment by jeremy-lq** [comment]
Can merge if it looks good to @derekwbrown.

File: content/agent/basic_agent_usage/windows.md [hunk 2]
```diff
@@ -46,6 +46,8 @@ The following configuration command line options are available when installing t
 | `PROXY_USER`      | String | If using a proxy, sets your proxy user. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
 | `PROXY_PASSWORD`  | String | If using a proxy, sets your proxy password. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                          |
 
+Note: If a valid `datadog.yaml` is found, that file takes precedence over all specified command-line options.
+
```
Comment (derekwbrown) [inline content/agent/basic_agent_usage/windows.md]: If a valid `datadog.yaml` is found, and has an API_KEY configured....

For better or worse, the conditional is whether an API key is set.

Comment (truthbk) [reply content/agent/basic_agent_usage/windows.md]: This was not the case when I tested it.... if the file was there, with no `API_KEY` the proxy settings did not get applied.

**Pull request merged**